jfr-connection/src/main/java/io/opentelemetry/contrib/jfr/connection/RecordingOptions.java

```diff
@@ -388,7 +388,7 @@ private static String validateDuration(Option option, String durationString) {
           case "m":
           case "h":
           case "d":
-            return Long.toString(value) + " " + units;
+            return Long.toString(value) + units;
```

Comment:
Actually, since the pattern is "([-+]?\\d+)\\s*(\\w*)", if there is a match, then group(1) is a number and there really is no need to do the parseLong.

Reply:
I rescind my comment here. The pattern allows +/- before the digits and a check is made to ensure the value is greater than or equal to zero. I think this is a valid check. I could change the pattern by dropping the +/-, but someone could (validly) put "+10s"